script.js

```diff
@@ -6,19 +6,18 @@ const booksContainer = document.querySelector('.books-container');
 const getLocalBooks = localStorage.getItem('books');
 const localBooks = JSON.parse(getLocalBooks);
 let booksCollection = getLocalBooks ? localBooks : [];
-
-function addBook(title, author) {
-  const book = {
-    title,
-    author,
-    id: parseInt((Math.random() * 1000000000), 10),
-  };
-  booksCollection.push(book);
-  localStorage.setItem('books', JSON.stringify(booksCollection));
+class AddBook {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.id = parseInt((Math.random() * 1000000000), 10);
+  }
 }
 
 function handleChange() {
-  addBook(bookTitle.value, bookAuthor.value);
+  const book = new AddBook(bookTitle.value, bookAuthor.value);
+  booksCollection.push(book);
+  localStorage.setItem('this', JSON.stringify(booksCollection));
 }
 
 function removeBook() {
```

Comment:
- [ ] <b>REQUIRED:</b> And all functions should be inside `Book` class. Kindly do that to comply with the project requirements. here is an example:
```js
class Books {
  constructor() {
    this.books = [];
  }

  addBook(title, author) {
  }

  removeBook(bookID) {
  }

  // getter method
  getBooks() {
  }
}

```